README.txt

```diff
@@ -5,15 +5,18 @@ running operation, providing a visual cue that processing is underway.
 
 The ProgressBar class manages the current progress, and the format of the line
 is given by a number of widgets. A widget is an object that may display
-differently depending on the state of the progress bar. There are three types
-of widgets:
- - a string, which always shows itself
+differently depending on the state of the progress bar.
 
- - a ProgressBarWidget, which may return a different value every time its
+There are mainly three types of widgets. They are :
+
+First  - a string, which always shows itself
```

Comment:
I think the formatting is good, but I don't think "First", "Second", "Third" are necessary. There's also another pull request changing this to markdown which may conflict with this one.